### DISCUSSION TITLE
Page gets stuck when using react component

### DISCUSSION THREAD

**Discussion by Archish27** [vidstack/player#1162] [General]
![image](https://github.com/vidstack/player/assets/9930610/e68db000-2d3a-40a2-b0e3-c961175960ff)
Please help page freezes when using MediaPlayer

**Comment by mihar-22** [comment]
There's no information here to work off at all to determine the issue. The player works fine, could be something in your setup. Closing this, open an issue, provide more info and preferably a [repro](https://github.com/vidstack/player/blob/main/.github/REPRO.md).
